src/tools/r2rdump/EHInfo.cs

```diff
@@ -0,0 +1,234 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection.PortableExecutable;
+using System.Text;
+
+namespace R2RDump
+{
+    /// <summary>
+    /// If COR_ILMETHOD_SECT_HEADER::Kind() = CorILMethod_Sect_EHTable then the attribute
+    /// is a list of exception handling clauses.  There are two formats, fat or small
+    /// </summary>
+    public enum CorExceptionFlag
+    {
+        COR_ILEXCEPTION_CLAUSE_NONE,                    // This is a typed handler
+        COR_ILEXCEPTION_CLAUSE_OFFSETLEN = 0x0000,      // Deprecated
+        COR_ILEXCEPTION_CLAUSE_DEPRECATED = 0x0000,     // Deprecated
+        COR_ILEXCEPTION_CLAUSE_FILTER = 0x0001,         // If this bit is on, then this EH entry is for a filter
+        COR_ILEXCEPTION_CLAUSE_FINALLY = 0x0002,        // This clause is a finally clause
+        COR_ILEXCEPTION_CLAUSE_FAULT = 0x0004,          // Fault clause (finally that is called on exception only)
+        COR_ILEXCEPTION_CLAUSE_DUPLICATED = 0x0008,     // duplicated clause. This clause was duplicated to a funclet which was pulled out of line
+    }
+
+    /// <summary>
+    /// This class represents a single exception handling clause. It basically corresponds
+    /// to IMAGE_COR_ILMETHOD_SECT_EH_CLAUSE_FAT in
+    /// <a href="https://github.com/dotnet/coreclr/blob/master/src/inc/cordebuginfo.h">src\inc\corhdr.h</a>.
+    /// </summary>
+    public class EHClause
+    {
+        /// <summary>
+        /// Length of the serialized EH clause in the PE image.
+        /// </summary>
+        public const int Length = 6 * sizeof(int);
```

Comment:
In the constructor you use `sizeof(uint)` but here you're using `sizeof(int)`. Do these two need to be consistent?

Reply:
Well, the sizes are obviously equal but I'll fix this for the sake of consistency, thanks for pointing that out!